ui/plugin-lightbox/package.json

```diff
@@ -1,18 +1,60 @@
 {
   "name": "@artalk/plugin-lightbox",
-  "version": "0.1.2",
+  "version": "0.2.1",
   "license": "MIT",
   "description": "The image lightbox plugin for artalk",
+  "type": "module",
   "main": "./dist/artalk-plugin-lightbox.js",
-  "module": "./dist/artalk-plugin-lightbox.es.js",
+  "module": "./dist/artalk-plugin-lightbox.mjs",
+  "types": "./dist/artalk-plugin-lightbox.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "serve": "vite preview",
-    "prepublish": "pnpm build",
-    "publish": "pnpm publish --access=public"
+    "serve": "vite preview"
   },
   "dependencies": {
     "artalk": "workspace:^"
```

Comment:
I think this should be better announced as peer to avoid duplicated artalk versions while users are installing any plugin while do not have latest artalk.at meantime.